### PR TITLE
Admin more commands

### PR DESCRIPTION
**Pull request by NoCatt**

### CODE REVIEW

File: mods/Fifty.ServerUtilities/mod/scripts/vscripts/fsu_core.nut [hunk 1]
```diff
@@ -147,6 +147,18 @@ void function FSU_PrintFormattedList( entity player, array< string > list, int p
...
 
+/**
+ * Returns the given array as a sting
+ * @param args Array of stings that will be
+ * @param separator optional argument of string that will separate all words
+*/
+string function FSU_ArrayToString( array< string > args, string separator = " " ) {
+	string result = ""
+	foreach( string s in args )
+		result += s + seperator
```
Comment (F1F7Y) [inline mods/Fifty.ServerUtilities/mod/scripts/vscripts/fsu_core.nut]: Doesn't compile

**Review by F1F7Y — CHANGES_REQUESTED** [review]

File: mods/Fifty.ServerAdmin/mod/scripts/vscripts/fsa_core.nut [hunk 2]
```diff
@@ -126,6 +190,239 @@ bool function FSA_IsAdmin( entity player ) {
...
+
+  if(args.len() <= 0)
+    thread FSU_C_Reload_thread(5.0)
+  else
+    thread FSU_C_Reload_thread(args[0].tofloat())
+}
+
+void function FSU_C_Reload_thread(float time){
+  while(time > 0){
+    Chat_ServerBroadcast("The server will reload in "+ time)
```
Comment (F1F7Y) [inline mods/Fifty.ServerAdmin/mod/scripts/vscripts/fsa_core.nut]: Use `FSU_ChatBroadcast`

File: mods/Fifty.ServerUtilities/mod/scripts/vscripts/fsu_core.nut (hunk 1 above)
Comment (NoCatt) [reply mods/Fifty.ServerUtilities/mod/scripts/vscripts/fsu_core.nut]: oh oops I think I need to add brakets

File: mods/Fifty.ServerAdmin/mod/scripts/vscripts/fsa_core.nut (hunk 2 above)
Comment (NoCatt) [reply mods/Fifty.ServerAdmin/mod/scripts/vscripts/fsa_core.nut]: what about the RUI messages do I swap them too?

Comment (NoCatt) [reply mods/Fifty.ServerAdmin/mod/scripts/vscripts/fsa_core.nut]: okay swaped all of them

**Pull request merged**